java/org/apache/tomcat/util/net/openssl/panama/OpenSSLContext.java

```diff
@@ -503,7 +504,7 @@ public void init(KeyManager[] kms, TrustManager[] tms, SecureRandom sr) throws K
                 }
             }
             // Check if the ciphers have been changed from the defaults
-            if (maxTlsVersion >= TLS1_3_VERSION() && (sslHostConfig.getCiphers() != SSLHostConfig.DEFAULT_TLS_CIPHERS)) {
+            if (maxTlsVersion >= TLS1_3_VERSION() && (!Objects.equals(sslHostConfig.getCiphers(), SSLHostConfig.DEFAULT_TLS_CIPHERS))) {
```

Comment:
This one is at least not-harmful but unnecessary. The default value of Ciphers is DEFAULT_TLS_CIPHERS (exactly, as in "the exact same object reference").

Reply:
The is supposed to check if the value was explicitly configured, even if it stayed the same as the default in the end. This is for TLS 1.3 cipher suite stuff (I'm not an expert on that).